rascal-textmate-core/src/test/sh/lang/textmate/conversion-tests.bat

```diff
@@ -0,0 +1,3 @@
+npx vscode-tmgrammar-test ^
```

Comment:
looks like the bat is the exact same.

how about we move it to `package.json` in the run block. and in rascal we just use `npm run conversion-tests` ?

Reply:
Fixed:

  - Call `npx` directly from Rascal instead of via the scripts
  - Remove the scripts